product_docs/docs/eprs/7/05_smr_operation/01_prerequisites/04_preparing_pub_database.mdx

```diff
@@ -316,7 +316,7 @@ GRANT INSERT  ON _edb_replicator_pub.rrep_tx_seq TO newuser;
 GO
```

Comment:
Since we are also not officially supporting SQL Server 2008 anymore, we should also remove the above reference and commands for it.